config/dr-cluster/rbac/role.yaml

```diff
@@ -8,24 +8,19 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - events
+  - secrets
```

Comment:
This bit of magic is to avoid errors like:
```
E0823 19:54:10.378459       1 reflector.go:138] pkg/mod/k8s.io/client-go@v0.21.3/tools/cache/reflector.go:167: Failed to watch *v1.Secret: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:ramen-system:ramen-dr-cluster-operator" cannot list resource "secrets" in API group "" at the cluster scope
```

Although we do not Own/Watch/For any of these in the sdk.

The issue seems to be covered here `https://github.com/kubernetes-sigs/kubebuilder/issues/1366` and needs better understanding before this is removed.

Overall, with these changes the controllers behave as required, and the bundles are created with the required RBACs as well. (moving out of draft)